File: app/src/main/java/com/softaai/designpatterns/java/creational/builder/meal/item/ColdDrink/Pepsi.java
package com.softaai.designpatterns.java.creational.builder.meal.item.ColdDrink;

/**
 * Created by Amol Pawar on 01-04-2019.
 * softAai Apps
 */
public class Pepsi extends ColdDrink {
    @Override
    public String name() {
        return "Pepsi";
    }

    @Override
    public float price() {
        return 35.0f;
    }
}

File: app/src/main/java/com/softaai/designpatterns/kotlin/creational/builder/meal/item/ColdDrink/Pepsi.kt
package com.softaai.designpatterns.kotlin.creational.builder.meal.item.ColdDrink


/**
 * Created by Amol Pawar on 01-04-2019.
 * softAai Apps
 */
class Pepsi : ColdDrink() {
    override fun price(): Float {
        return 35.5f
    }

    override fun name(): String {
        return "Pepsi"
    }
}